tests/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixerTest.php

```diff
@@ -45,6 +45,9 @@ public function provideFixCases()
      * Description.
      *
      * @param string|null $str   some string
+     * @param string $ip         IPv4 is not lowercased
```

Comment:
I wouldn't lowercase non-words, simple check would be that 2 first chars are letters, so no lowercasing for `T34` and `S§`. Please add tests